src/utils/userCache.js

```diff
@@ -0,0 +1,20 @@
+const userCache = {};
+
+const addUserToCache = (user) => {
+  const id = user.id;
+  if (userCache[id]) {
+    return;
+  }
+  userCache[id] = id;
+};
+
+const checkUserCache = (user) => {
+  if (userCache[user]) {
+    return true;
+  }
+  else {
+    return false;
+  }
+};
```

Comment:
you could replace with
```suggestion
const checkUserCache = (user) => {
  if (userCache[user]) {
    return true;
  }
  return false;
};
```
or
```suggestion
const checkUserCache = (user) => !!userCache[user]);
```

Reply:
lol i hate implicit `else` statements, makes the code awkward to read IMO, not married to it though. Ill commit the first change, i dont like the `!!`, its easier to read if its verbose. @Claudi0-V